index.html

```diff
@@ -760,9 +760,9 @@ <h2>
           Interaction with Web Application Manifest
         </h2>
         <p>
-          The <a data-cite="appmanifest"></a>Web Application Manifest
-          specification allows web applications to set the <a>document</a>'s
-          <a>default orientation</a>.
+          The <a data-cite="appmanifest">Web Application Manifest
```

Comment:
Just having `<a data-cite="appmanifest"></a>` should automatically fill in the specs title. So the tect after the closing `</a>` can probably go instead.